### ISSUE TITLE
Date of when I first joined is incorrect

### ISSUE THREAD

**Issue by HardstyleRaver2** [OvercastNetwork/Issues#1805]
My 3rd cake day is coming soon, so I went to check on my join date. Apparently, it says that I first joined the server on November of 2012 and not in April of 2012.

http://i.imgur.com/PMwpzQ1.png

**Comment by Brottweiler** [comment]
The join date is the first time you got a kill/death after join dates started being tracked. Are you sure you were playing at the time?

**Comment by HardstyleRaver2** [comment]
@Brottweiler I'm 100% sure. If you take a look at my first punishment (It has been appealed), you'll see that it has been issued on April 2012.

**Comment by mrapple** [comment]
We no longer take punishments into account. However, I think we're working on taking deaths into account

**Comment by nixter1029** [comment]
Yeah, this is majorly changed. My joined date use to be back in April of 2012 now it says Aug. Now people that joined after me, say they joined before me. Even your join date is the same as mine @mrapple

**Comment by itsmartin** [comment]
@nixter1029 same hetr
